crates/core/src/util/typed_prometheus.rs

```diff
@@ -121,6 +121,7 @@ impl_prometheusvalue_string!(
     Identity,
     Address,
     TransactionType,
+    SysCall,
```

Comment:
We've waffled a lot on calling this syscalls vs abicalls vs... what. And even in this PR you say SysCall here but wasm_abi_call elsewhere.

Let's pick one.

Reply:
I would argue it's really a syscall where SpacetimeDB is the OS. Happy to resolve the bikeshedding in a separate PR though.